notebooks/249-oneformer-segmentation/249-oneformer-segmentation.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<p>This description is not correct. Directly supported means that it is possible to provide model to core.read_model without any conversion step, but this scenario does not support PyTorch and conversion is only way to run pytorch models with openvino. Please remove this sentence.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>